modules/Kconfig.tfm

```diff
@@ -1,8 +1,23 @@
-# TF-M/PSA Related
+# Configuration for the TF-M Module
 
 # Copyright (c) 2019, 2020 Linaro Limited
+# Copyright (c) 2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+
+config TFM_BOARD
+	string
+	default "nordic_nrf/nrf9160dk_nrf9160" if BOARD_NRF9160DK_NRF9160NS
```

Comment:
Little late to the game, but why have all this knowledge here ?

This make users think they need to update this file in order to set `TFM_BOARD`.
For example, for the `nrf9160dk_nrf9160ns` I think it would be better to have this in:
https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf9160dk_nrf9160/Kconfig.defconfig
```
if BOARD_NRF9160DK_NRF9160NS
...

config TFM_BOARD
        default "nordic_nrf/nrf9160dk_nrf9160"

endif
```

Reply:
Discussion here: https://github.com/zephyrproject-rtos/zephyr/pull/30062#discussion_r524413356